task/test-github

```diff
@@ -179,7 +179,7 @@ class TestGitHub(unittest.TestCase):
 
     def testAllowlist(self):
         allowlist = self.api.allowlist()
-        self.assertTrue(len(allowlist) > 0)
```

Comment:
I wonder about these.  It seems to me that `pytest` has recently gained some magical ability to dissect normal `assert a > b` statements and get the values out...

But in any case, `assertGreater(a, b)` is definitely more correct than `assertTrue(a < b)`